client/a8c-for-agencies/components/layout/index.tsx

```diff
@@ -22,15 +22,7 @@ type Props = {
 	compact?: boolean;
 };
 
-function MainLayout( {
-	children,
-	className,
-	title,
-	wide,
-	withBorder,
-	compact,
```

Comment:
Now that we are standardizing the spacing for all our headers, there is no longer a need for the compact variant in our layout.